### PR TITLE
Regen with latest gir/gir-files

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: gdk4-wayland/src/auto/wayland_gl_context.rs [hunk 1]
```diff
@@ -13,6 +13,8 @@ glib::wrapper! {
     }
 }
 
+impl WaylandGLContext {}
```
Comment (bilelmoussaoui) [inline gdk4-wayland/src/auto/wayland_gl_context.rs:16]: I don't know what is causing this regression...

Comment (sdroege) [reply gdk4-wayland/src/auto/wayland_gl_context.rs:16]: It's since a while, I also don't know

File: gdk4/sys/Cargo.toml [hunk 2]
```diff
@@ -26,6 +26,9 @@ version = "4.3.2"
 [package.metadata.system-deps.gtk4.v4_6]
 version = "4.5"
```
Comment (sdroege) [inline gdk4/sys/Cargo.toml:27]: Should maybe be 4.6 at this point?

**Review by sdroege — COMMENTED** [review]
Seems ok to me otherwise

File: gdk4-wayland/src/auto/wayland_gl_context.rs (hunk 1 above)
Comment (bilelmoussaoui) [reply gdk4-wayland/src/auto/wayland_gl_context.rs:16]: I will investigate in the future. Opened an issue regarding that for now https://github.com/gtk-rs/gir/issues/1334

**Pull request merged**